ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,10 +1,33 @@
 #include "../PrjFSKext/kernel-header-wrappers/vnode.h"
 #import <XCTest/XCTest.h>
 #include "../PrjFSKext/KauthHandlerTestable.hpp"
+#include "../PrjFSKext/PerformanceTracing.hpp"
 #include "MockVnodeAndMount.hpp"
 
 using std::shared_ptr;
 
+#define KAUTH_RESULT_ALLOW    (1)
+#define KAUTH_RESULT_DENY     (2)
+#define KAUTH_RESULT_DEFER    (3)
+
+proc_t vfs_context_proc(vfs_context_t ctx)
+{
+    return NULL;
+}
+
+extern "C" int proc_pid(proc_t);
+int proc_pid(proc_t)
+{
+    return 1;
+}
+
+extern "C" void proc_name(int pid, char * buf, int size);
+char * procname;
+void proc_name(int pid, char * buf, int size)
+{
+    buf = procname;
+}
+
```

Comment:
I don't love having the definitions and 3 functions here, but i'm reluctant to move them unless we think we'll use them elsewhere.  
If folks think we should move please also suggest a place/filename.

Reply:
> The thing to bear in mind with having them defined here is that they equally apply to other test suites because they're all linked into a single bundle.

Thats a good point to remember.  Moving to MockProc.  We'll get more specific / move in the future if needbe.